extensions/ql-vscode/src/variant-analysis/run-remote-query.ts

```diff
@@ -59,9 +60,11 @@ interface GeneratedQueryPack {
  */
 async function generateQueryPack(
   cliServer: CodeQLCliServer,
-  queryFile: string,
+  qlPackDetails: QlPackDetails,
```

Comment:
This function handles when the queries are in an existing pack as well as when they are not. In that case would the `QlPackDetails` be undefined, or would we still pass this object but with some of the fields within it empty? It's description says "details about the original QL pack" which might not be incorrect in the "no existing pack" case.

Reply:
I'm imagining `QlPackDetails` to still be defined (with some empty fields), so ideally I would prefer a different name for it, but I haven't been able to come up with a better name. Perhaps one will more naturally come up as we make the changes (or once we're done with them).

Reply:
A better name would definitely be nice, but I also can't think of anything right now. Happy to go ahead and address it later when we have a better idea of things.